src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -776,34 +790,34 @@ private AIState completeBuild()
 
         String schematicName = job.getSchematic().getName();
         LanguageHandler.sendPlayersLocalizedMessage(EntityUtils.getPlayersFromUUID(world, worker.getColony().getPermissions().getMessagePlayers()),
-                "entity.builder.messageBuildComplete",
-                schematicName);
+                                                    "entity.builder.messageBuildComplete",
+                                                    schematicName);
 
         WorkOrderBuild wo = job.getWorkOrder();
-        if(wo != null)
+        if (wo != null)
         {
-            if(!(wo instanceof WorkOrderBuildDecoration))
+            if (!(wo instanceof WorkOrderBuildDecoration))
             {
                 AbstractBuilding building = job.getColony().getBuilding(wo.getBuildingLocation());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'building' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)